hermes-management/src/main/java/pl/allegro/tech/hermes/management/domain/subscription/consumergroup/command/ScheduleConsumerGroupToDeleteCommand.java

```diff
@@ -0,0 +1,53 @@
+package pl.allegro.tech.hermes.management.domain.subscription.consumergroup.command;
+
+import java.time.Instant;
+import pl.allegro.tech.hermes.api.SubscriptionName;
+import pl.allegro.tech.hermes.management.domain.dc.DatacenterBoundRepositoryHolder;
+import pl.allegro.tech.hermes.management.domain.dc.RepositoryCommand;
+import pl.allegro.tech.hermes.management.domain.subscription.consumergroup.ConsumerGroupAlreadyScheduledToDeleteException;
+import pl.allegro.tech.hermes.management.domain.subscription.consumergroup.ConsumerGroupToDelete;
+import pl.allegro.tech.hermes.management.domain.subscription.consumergroup.ConsumerGroupToDeleteRepository;
+
+public class ScheduleConsumerGroupToDeleteCommand
+    extends RepositoryCommand<ConsumerGroupToDeleteRepository> {
+  private final SubscriptionName subscriptionName;
+  private final Instant requestedAt;
+
+  private ConsumerGroupToDelete consumerGroupToDelete;
+
+  public ScheduleConsumerGroupToDeleteCommand(
+      SubscriptionName subscriptionName, Instant requestedAt) {
+    this.subscriptionName = subscriptionName;
+    this.requestedAt = requestedAt;
+  }
+
+  @Override
+  public void backup(DatacenterBoundRepositoryHolder<ConsumerGroupToDeleteRepository> holder) {}
+
+  @Override
+  public void execute(DatacenterBoundRepositoryHolder<ConsumerGroupToDeleteRepository> holder) {
+    consumerGroupToDelete =
```

Comment:
maybe we don't have to have this mutable property? for rollback we can get datacenter name from holder and we don't need to cache this